finality-aleph/src/party/mod.rs

```diff
@@ -478,14 +481,17 @@ where
                 },
                 Some(_) = async {
                     match &mut start_next_session_network {
-                        Some(start_next_session_network) => {
-                            start_next_session_network.await;
+                        Some(notification) => {
+                            if let Err(e) = notification.await {
+                                error!(target: "aleph-party", "Error with subscription {:?}", e);
```

Comment:
This should only be a `warn`, something went wrong, but we can go on.

Reply:
done